packages/components/select-v2/src/select.vue

```diff
@@ -340,12 +348,29 @@ export default defineComponent({
   ],
 
   setup(props, { emit }) {
-    const API = useSelect(props, emit)
+    const modelValue = computed(() =>
+      props.multiple
+        ? isArray(props.modelValue)
+          ? props.modelValue
+          : []
+        : isArray(props.modelValue)
+        ? undefined
+        : props.modelValue
+    )
```

Comment:
I think this can be simplified as:

```ts
const { modelValue, multiple } = props;
const fallback = multiple ? [] : undefined;
// When it is array, we check if this is multi-select.
// Based on the result we get
if (isArray(props.modelValue) {
  return multiple ? modelValue : fallback;
}
return modelValue;
```
I'd also like to suggest that we should not nest ternary operator inside a ternary operator just like the code here.

Reply:
Thanks for your advice. 👍